aggregate_spatial.json

```
@@ -0,0 +1,130 @@
+{
+    "id": "aggregate_spatial",
+    "summary": "Compute zonal statistics for a vector data cube",
```

```suggestion
    "summary": "Compute zonal statistics for geometries",
```
GeoJSON is a potential argument, but not a data cube itself. So I would propose to make it more generic. Either "for vector data" or "for geometries".